contracts/StableJoeStaking.sol

```diff
@@ -299,16 +309,16 @@ contract StableJoeStaking is Initializable, OwnableUpgradeable {
         uint256 _currRewardBalance = _token.balanceOf(address(this));
         uint256 _rewardBalance = _token == joe ? _currRewardBalance.sub(internalJoeBalance) : _currRewardBalance;
 
-        uint256 _totalJoe = internalJoeBalance;
```

Comment:
This line may seem useless, but in fact it's some gas optimization, when you use a var more than once it costs less gas to declare a new var copying that value. As we're using `updateReward` inside a loop, we need to take care of gas, so I'd be in favor of keeping that line and using in the next 2 calls

Reply:
ahh I see, that makes sense! fixed